data/channels.csv

```diff
@@ -21513,145 +21513,142 @@ NST.ru,NST,НСТ;Настоящее Страшное Телевидение,,VG
-NTM.ru,NTM,НТМ,,,RU,,,c/RU,rus,,FALSE,,,,https://ntm13.ru/,https://i.imgur.com/AH50pGT.png
+NTK.kz,NTK,НТК,,,KZ,,Almaty,c/KZ,kaz;rus,general,FALSE,,,,http://www.ntk.kz/,https://i.imgur.com/hHSCH4L.png
+NTKTV.ua,NTK TV,TV,,,UA,,Kolomyia,c/UA,ukr,general,FALSE,,,,https://www.ntktv.ua/,https://www.ntktv.ua/bitrix/templates/ntk_copy/images/logo.png
+NTM.ru,NTM,НТМ,,NTM RUS OOO,RU,,Moscow,c/RU,rus,general,FALSE,,,,https://ntm13.ru/,https://i.imgur.com/AH50pGT.png
 NTN.it,NTN,Nuova TV Nazionale,,Giorgio Mendella,IT,,Pistoia,c/IT,ita,general,FALSE,,,,https://www.nuovatvnazionale.it/,https://i.imgur.com/AW5fJwa.png
-NTN.mn,NTN,,,,MN,,,c/MN,mon,,FALSE,,,,,https://i.imgur.com/56YJg8G.png
-NTN.ua,NTN,НТН,,Ukrainian Media Resource;Mindon Holdings,UA,,Kyiv,c/UA,ukr,,FALSE,2004-11-01,,,http://ntn.ua/,https://i.imgur.com/AmyCNJv.png
-NTN24.co,NTN24,,,RCN Televisión,CO,,,r/INT,spa,news,FALSE,,,,https://www.ntn24.com/,https://i.imgur.com/UoKmSAP.png
-NTRTV.tr,NTR TV,,,,TR,,,c/TR,eng,,FALSE,,,,https://ntrtv.com.tr/,https://i.imgur.com/6DnwS5i.png
+NTN.mn,NTN,,,,MN,,Ulaanbaatar,c/MN,mon,news,FALSE,,,,,https://i.imgur.com/56YJg8G.png
+NTN.ua,NTN,НТН,,Ukrainian Media Resource;Mindon Holdings,UA,,Kyiv,c/UA,ukr,general,FALSE,2004-11-01,,,http://ntn.ua/,https://i.imgur.com/AmyCNJv.png
+NTN24.co,NTN24,,,RCN Televisión,CO,,Bogota,r/INT,spa,news,FALSE,2008-11-03,,,https://www.ntn24.com/,https://i.imgur.com/UoKmSAP.png
+NTRTV.tr,NTR TV,Afroturk TV,,,TR,,Ankara,c/TR,eng,culture,FALSE,,,,https://ntrtv.com.tr/,https://i.imgur.com/6DnwS5i.png
 NTS.kg,NTS,НТС;ТРК НТС,,NTS,KG,,Bishkek,c/KG,kir;rus,general,FALSE,,,,http://nts.kg/,https://i.ibb.co/hCf7v4D/nts.jpg
-NTS.ru,NTS,НТС,,,RU,,Sevastopol,c/RU,rus,,FALSE,,,,https://nts-tv.com/,https://i.imgur.com/skkDR7q.png
+NTS.ru,NTS,НТС,,,RU,,Sevastopol,c/RU,rus,general,FALSE,,,,https://nts-tv.com/,https://i.imgur.com/skkDR7q.png
 NTSTV.md,NTS TV,,,,MD,,,c/MD,rus,general,FALSE,,,,https://www.nts-tv.md/,https://i.imgur.com/LL2gMNF.png
 NTTV.us,NTTV,New Trier Television,,,US,US-IL,Northfield,s/US-IL,eng,education,FALSE,,,,https://nths.mediaspace.kaltura.com/media/NTTV/1_i3yjnteq,https://i.imgur.com/q1yWTvC.png
-NTV.ae,NTV,,,,AE,,,c/AE,eng,,FALSE,,,,http://www.ntvuae.tv/,https://i.imgur.com/t59sqk1.png
+NTV.ae,NTV,,,,AE,,Dubai,c/AE,eng,general,FALSE,,,,http://www.ntvuae.tv/,https://i.imgur.com/t59sqk1.png
 ntv.al,ntv,Neser TV,,,AL,,Tirana,c/AL,sqi,news,FALSE,1998-01-01,,,https://ntv.al/,https://i.imgur.com/gtZLzxq.png
-ntv.at,n-tv,,,RTL Group,AT,,,c/AT,deu,,FALSE,1992-11-30,,,https://www.n-tv.de/,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/N-tv_logo-september2017.svg/512px-N-tv_logo-september2017.svg.png
-NTV.bd,NTV,,,,BD,,,c/BD,ben,,FALSE,,,,https://www.ntvbd.com/,https://i.imgur.com/l75bDTx.png
+ntv.de,n-tv,,,RTL Group,DE,,Cologne,c/DE;c/CH;c/AT,deu,general,FALSE,1992-11-30,,,https://www.n-tv.de/,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/N-tv_logo-september2017.svg/512px-N-tv_logo-september2017.svg.png

```

Comment:
Why replace `ntv.at` with `ntv.de`?

Reply:
The replacement was needed due to the fact that the channel was never Austrian to begin with, and it always was German. That's what AP told me.
![image](https://github.com/user-attachments/assets/1fa9551a-b16f-4539-a49f-753d9e922b13)

Though https://oesterreichprogramm.at/ntv-austria/ seems to indicate there's a branch in Austria... since pretty recently (on satellite since August of this year).

Reply:
![Screenshot_2024-12-25-02-36-38-73_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/e8d95df3-b191-4786-9ce3-b5ca1153dd65)

Reply:
I stand corrected: add the Austrian entry back in.